lua/telescope/make_entry.lua

```diff
@@ -1130,5 +1130,57 @@ function make_entry.gen_from_git_status(opts)
   end
 end
 
+function make_entry.gen_from_jumplist(opts)
+  opts = opts or {}
+  opts.tail_path = get_default(opts.tail_path, true)
+
+  local displayer = entry_display.create {
+    separator = "▏",
+    items = {
+      { width = 10 },
+      { remaining = true },
+    }
+  }
+
+  local make_display = function(entry)
+    local filename
+    if not opts.hide_filename then
+      filename = entry.filename
+      if opts.tail_path then
+        filename = utils.path_tail(filename)
+      elseif opts.shorten_path then
+        filename = utils.path_shorten(filename)
+      end
+    end
+
+    local line_info = {table.concat({entry.lnum, entry.col}, ":"), "TelescopeResultsLineNr"}
+
+    return displayer {
+      line_info,
+      filename,
+    }
+  end
+
+  return function(entry)
+    local filename = entry.filename or vim.api.nvim_buf_get_name(entry.bufnr)
```

Comment:
Just tested it. I am getting: `Invalid buffer id: 1` here if i had startify open before.
So: `nvim` -> `:Telescope find_files` -> `open file` -> `:Telescope jumplist` (error)
If i do: `nvim file` -> `:Telescope jumplist` works great

We should do
```lua
if not vim.api.nvim_buf_is_valid(entry.bufnr) then
  return
end
```
before this line. Thoughts?

Reply:
Otherwise works great :) thanks

Reply:
Added this to the commit.

I personally don't use startify, and haven't been able to reproduce a scenario where this makes/breaks anything. I think its OK, unless there is some other sort of verification we can do.